rpm/native_db.go

```diff
@@ -44,8 +45,13 @@ func getDBObjects[T ObjectResponse](ctx context.Context, sys fs.FS, db foundDB,
 	switch db.Kind {
 	case kindSQLite:
 		r, err := sys.Open(path.Join(db.Path, `rpmdb.sqlite`))
-		if err != nil {
-			return nil, fmt.Errorf("rpm: error reading sqlite db: %w", err)
+		switch {
+		case errors.Is(err, nil):
+		case errors.Is(err, fs.ErrNotExist):
+			zlog.Warn(ctx).Err(err).Msg("rpm: unable to open sqlite db")
```

Comment:
can you confirm you see this logged when you run the test? Just want to make sure this is the expected error (not exist)